### ISSUE TITLE
[Question] releases to crates.io

### ISSUE THREAD

**Issue by replicadse** [xliiv/fui#127]
**TLDR;**
I came across this crate a while ago and I have implemented support to it into one of the tools I built.
A few days ago, you merged a PR from @gyscos for supporting cursive 0.16. Now this is nice since I am in fact able to specify a Cargo.toml dependency to this crate by specifying the repo+ref (commit).
The only issue with that is that I can now not publish that crate to crates.io since it does not allow these kinds of dependencies.
The current version of fui that is on crates.io is quite old from July 2019 and references cursive 0.11.
I am referencing this: https://github.com/replicadse/complate/blob/master/Cargo.toml

**Question:**
Do you plan to release a new version of fui to crates.io anytime soon? This would enable me to update the dependencies of the packages that reference fui v1.0.1 (and transitively cursive v0.11). I was not able to find any todo or plan to do so or am I missing something obvious?

Thanks in advance!

PS: awesome crate!

**Comment by xliiv** [comment]
I am glad you like it.

> Do you plan to release a new version of fui to crates.io anytime soon?

Sure, I'll push it this week? Does it work for you?

**Comment by replicadse** [comment]
That would be amazing! Thank you very much.

**Comment by xliiv** [comment]
@replicadse, version 2.0.0 released (breaking changes - major upgrade), see https://crates.io/crates/fui

Enjoy :)

**Comment by replicadse** [comment]
Thanks a lot! I was now able to release a new version.
https://crates.io/crates/complate/0.9.0-beta.2

**Comment by xliiv** [comment]
Good to hear and congratulations @replicadse
